src/modules/markets/markets.service.ts

```diff
@@ -15,7 +15,7 @@ export class MarketsService {
     let spResults = await this.marketsRepository.query('CIS_USP_READ_MARKETS');
```

Comment:
just a thought, should/could this be a table? rather than SP?

Reply:
because in that case we could use something like
```
    let filter = {};
    if (active === 'Y') {
      filter = { active: 'Y' };
    } else if (active === 'N') {
      filter = { active: 'N' };
    }

    const response = await this.marketsRepository.find({ where: [filter] });
```